### ISSUE TITLE
Python opm package should expose __version__

### ISSUE THREAD

**Issue by berland** [OPM/opm-common#1881]
According to PEP396, the opm package on pypi should expose the `__version__` attribute.

**Comment by nomadcoder0101** [comment]
@berland Hi Havard, I am trying to pip install opm library, but it results into an error as "Could not find the version that satisfies the requirement." 
Since I am new to this, can you please let me know steps for installing OPM library.

**Comment by berland** [comment]
Hi. The mailing list should be used for support questions, https://opm-project.org/?page_id=358.

On my Linux machine, `pip install opm` works perfectly.

**Comment by alfbr** [comment]
Indeed, the mailinglist is well suited. Please remember to subscribe to the list before posting. Otherwise your message will not go through.

**Comment by nomadcoder0101** [comment]
@berland Hi Havard, thank you for the reply. I have subscribed to the mailing list.
I was trying it on windows machine. Is it like the library is not yet supported on windows.

**Comment by alfbr** [comment]
> @berland Hi Havard, thank you for the reply. I have subscribed to the mailing list.
> I was trying it on windows machine. Is it like the library is not yet supported on windows?

You should use WSL to run it on Windows, see: https://docs.microsoft.com/en-us/windows/wsl/install-win10
